test/suites/backup.sh

```diff
@@ -784,6 +797,14 @@ test_backup_volume_export_with_project() {
   lxc storage volume get "${custom_vol_pool}" testvol/test-snap0 user.foo | grep -Fx "test-snap0"
   lxc storage volume get "${custom_vol_pool}" testvol/test-snap1 user.foo | grep -Fx "test-snap1"
 
+  # Check if the imported volume and its snapshots have a new UUID.
```

Comment:
should they?

Reply:
Yes, see https://github.com/canonical/lxd/pull/12840#discussion_r1489629279